content/blog/kubernetes-monitoring-using-prometheus-and-grafana-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Adding a monitoring stack to a Kubernete cluster using Prometheus and
+  Grafana in HPE GreenLake for Private Cloud Enterprise
+date: 2024-01-04T09:08:32.582Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - Prometheus
+  - Grafana
+  - Kubernetes
+  - Kubernetes monitoring
+  - HPE GreenLake for Private Cloud Enterprise
+---
+### Introduction
+
+[HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html), one of the HPE GreenLake cloud services available on the HPE GreenLake for Private Cloud Enterprise, allows customers to create a Kubernetes (K8s) cluster, view details about existing clusters, and deploy containerized applications to the cluster. It provides an enterprise-grade container management service using open source K8s. 
```

Comment:
[HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html) ("containers service"), one of the ....